src/snowflake/connector/s3_storage_client.py

```diff
@@ -92,6 +93,9 @@ def __init__(
     def transfer_accelerate_config(
         self, use_accelerate_endpoint: bool | None = None
     ) -> bool:
+        # accelerate cannot be used in China and us government
+        if "S3China" in self.location_type:
```

Comment:
is this case sensitive, also just curious where do you find out this constant value

Reply:
Got this information from storage team as well